src/transformState.js

```diff
@@ -5,7 +5,28 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
```

Comment:
You should practice destructuring here

Reply:
discussed with Denys, agreed that destructuring should not be used here.
PR approved!